specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-02-02-preview/managedClusters.json

```diff
@@ -5618,11 +5924,157 @@
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "azure",
+                "description": "Use the Azure CNI network plugin. See [Azure CNI (advanced) networking](https://docs.microsoft.com/azure/aks/concepts-network#azure-cni-advanced-networking) for more information."
+              },
+              {
+                "value": "kubenet",
+                "description": "Use the Kubenet network plugin. See [Kubenet (basic) networking](https://docs.microsoft.com/azure/aks/concepts-network#kubenet-basic-networking) for more information."
+              },
+              {
+                "value": "none",
+                "description": "Do not use a network plugin. A custom CNI will need to be installed after cluster creation for networking functionality."
+              }
+            ]
+          },
+          "description": "Network plugin used for building the Kubernetes network."
+        },
+        "networkPolicy": {
+          "type": "string",
+          "enum": [
+            "calico",
+            "azure"
+          ],
+          "x-ms-enum": {
+            "name": "NetworkPolicy",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "calico",
+                "description": "Use Calico network policies. See [differences between Azure and Calico policies](https://docs.microsoft.com/azure/aks/use-network-policies#differences-between-azure-and-calico-policies-and-their-capabilities) for more information."
+              },
+              {
+                "value": "azure",
+                "description": "Use Azure network policies. See [differences between Azure and Calico policies](https://docs.microsoft.com/azure/aks/use-network-policies#differences-between-azure-and-calico-policies-and-their-capabilities) for more information."
+              }
+            ]
+          },
+          "description": "Network policy used for building the Kubernetes network."
+        },
+        "networkMode": {
+          "type": "string",
+          "enum": [
+            "transparent",
+            "bridge"
+          ],
+          "x-ms-enum": {
+            "name": "networkMode",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "transparent",
+                "description": "No bridge is created. Intra-VM Pod to Pod communication is through IP routes created by Azure CNI. See [Transparent Mode](https://docs.microsoft.com/azure/aks/faq#transparent-mode) for more information."
+              },
+              {
+                "value": "bridge",
+                "description": "This is no longer supported"
+              }
+            ]
+          },
+          "title": "The network mode Azure CNI is configured with.",
+          "description": "This cannot be specified if networkPlugin is anything other than 'azure'."
+        },
+        "loadBalancerSku": {
```

Comment:
is it possible to reference some predefined enum.

Reply:
yup